ksqldb-common/pom.xml

```diff
@@ -125,7 +125,6 @@
                 <configuration>
                     <compilerArgs>
                         <arg>-Xlint:all,-serial</arg>
-                        <arg>-Werror</arg>
```

Comment:
instead, can we replace this with `#{compile.warnings-flag}`? We'd like to keep the `-Werror` on ksql builds, and we have setup upstream builds to set this to false. Not sure why it wasn't updated in this pom! (all the other modules have it set like that)

Reply:
@agavra Are there several places where this change needs to happen?  I count 10 below?

```
$ find . -name pom.xml -exec egrep "compile.warnings-flag|Werror" {} \; -ls
                        <arg>${compile.warnings-flag}</arg>
21763229        8 -rw-r--r--    1 rdagostino       staff                2907 Jun 22 07:40 ./ksqldb-streams/pom.xml
                        <arg>-Werror</arg>
21762981        8 -rw-r--r--    1 rdagostino       staff                2560 Jun 22 07:40 ./ksqldb-rest-model/pom.xml
                        <arg>-Werror</arg>
21772257        8 -rw-r--r--    1 rdagostino       staff                3969 Jun 22 07:46 ./ksqldb-udf-quickstart/target/classes/archetype-resources/pom.xml
                        <arg>-Werror</arg>
21763454        8 -rw-r--r--    1 rdagostino       staff                3973 Jun 22 07:40 ./ksqldb-udf-quickstart/src/main/resources/archetype-resources/pom.xml
                        <arg>-Werror</arg>
21747039       16 -rw-r--r--    1 rdagostino       staff                5262 Jun 22 07:40 ./ksqldb-cli/pom.xml
                        <arg>-Werror</arg>
21763428        8 -rw-r--r--    1 rdagostino       staff                3387 Jun 22 07:40 ./ksqldb-tools/pom.xml
            <arg>${compile.warnings-flag}</arg>
21747016       16 -rw-r--r--    1 rdagostino       staff                5892 Jun 22 07:40 ./ksqldb-benchmark/pom.xml
                        <arg>-Werror</arg>
21762588       16 -rw-r--r--    1 rdagostino       staff                7926 Jun 22 07:40 ./ksqldb-rest-app/pom.xml
        <compile.warnings-flag>-Werror</compile.warnings-flag>
                        <arg>${compile.warnings-flag}</arg>
21763653       48 -rw-r--r--    1 rdagostino       staff               23497 Jun 22 07:40 ./pom.xml
                        <arg>${compile.warnings-flag}</arg>
21762394       16 -rw-r--r--    1 rdagostino       staff                4353 Jun 22 07:40 ./ksqldb-parser/pom.xml
            <arg>-Werror</arg>
21748462       16 -rw-r--r--    1 rdagostino       staff                5848 Jun 22 07:40 ./ksqldb-functional-tests/pom.xml
                        <arg>-Werror</arg>
21748156        8 -rw-r--r--    1 rdagostino       staff                3835 Jun 22 07:40 ./ksqldb-examples/pom.xml
                        <arg>${compile.warnings-flag}</arg>
21763532       16 -rw-r--r--    1 rdagostino       staff                4501 Jun 22 07:40 ./ksqldb-version-metrics-client/pom.xml
                        <arg>-Werror</arg>
21763120       16 -rw-r--r--    1 rdagostino       staff                4445 Jun 22 07:40 ./ksqldb-serde/pom.xml
                         <arg>${compile.warnings-flag}</arg>
21748208        8 -rw-r--r--    1 rdagostino       staff                3403 Jun 22 07:40 ./ksqldb-execution/pom.xml
            <arg>-Werror</arg>
21762948        8 -rw-r--r--    1 rdagostino       staff                2658 Jun 22 07:40 ./ksqldb-rest-client/pom.xml
                        <arg>${compile.warnings-flag}</arg>
21747423       16 -rw-r--r--    1 rdagostino       staff                6444 Jun 22 07:40 ./ksqldb-engine/pom.xml
$
```

Reply:
other than the `ksqldb-udf-quickstart` they can probably all be changed! good catch